shared/fs/filepreview/index.stories.tsx

```diff
@@ -113,10 +115,17 @@ const store = {
     pathItems:
       // @ts-ignore
       I.Map([
```

Comment:
wanna de-immutable this? / also wanna un-ignore it if that turns out to be easy?

Reply:
oops missed this. Thanks for calling out